src/components/Functions/PoseCalibration.svelte

```diff
@@ -0,0 +1,80 @@
+<script>
+    import {writable} from "svelte/store";
+    import {openSidecar} from "../../utils/sidecar";
+    import ToastStore from "../../stores/toast";
+    import Select from "../Input/Select.svelte";
+    import Text from "../Input/Text.svelte";
+    import SuspenseButton from "../Input/Button/SuspenseButton.svelte";
+
+    const state = writable({
+            form: {
+                calibrationTimer: 5,
+                rightHand: true,
+            },
+            timer: null,
+            calibrating: false,
+            loading: false,
+    });
+
+    const beginCalibration = async () => {
+        $state.calibrating = true;
+        $state.timer = $state.form.calibrationTimer;
+
+        try {
+            await openSidecar('sidecar', 'functions_autocalibrate', {
```

Comment:
Just a minor complaint, the commands being called don't match the name of the file (or display text). Feel free to ignore

Reply:
fixed